pymc4/random_variables/continuous.py

```diff
@@ -246,7 +246,7 @@ def _base_dist(self, sigma, *args, **kwargs):
 
 class HalfStudentT(RandomVariable):
     r"""
-    Half Student's T log-likelihood
+    Half Student's T distribution.
```

Comment:
I hate to nitpick but these are technically random variables, not distributions...

Reply:
Also nit: mixing style of `r"""` and `R"""`

Reply:
Fixed the nits!